pages/components/Alerts/alerts.css

```diff
@@ -1,7 +1,7 @@
 @import url(../../../css/main.css);
 
 .alert {
-  width: max-content;
+  /* width: max-content; */
   background-color: aquamarine;
```

Comment:
you could use variables instead of direct colours, as it will allow you to manage all colours on one single page.